frontend/src/components/Button.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import styled from 'styled-components'
+
+interface StyledButtonProps {
+  primary?: boolean
+  danger?: boolean
+}
+
+const ButtonWrapper = styled.button`
+  background: ${(props: StyledButtonProps) =>
+    props.primary ? 'grey' : 'white'};
+  color: ${(props: StyledButtonProps) => (props.primary ? 'white' : 'grey')};
+  border-radius: 5px;
+  height: 3.125 em;
```

Comment:
There's a typo that's causing the height to be invalid. Should be "3.125em" without a space